ext_mongodb.php

```diff
@@ -210,7 +210,7 @@ function($value) {
 <<__NativeData("MongoDBDriverManager")>>
 class Manager {
 	<<__Native>>
-	public function __construct(string $dsn = "mongodb://127.0.0.1/", array $options = array(), array $driverOptions = array());
+	public function __construct(string $dsn = "", array $options = array(), array $driverOptions = array());
```

Comment:
And I'd have to change things into a variant in the c++ code too... I'm going to say "not worth messing with".